main.scss

```diff
@@ -4,11 +4,15 @@ $o-video-is-silent: true !default;
 @import "o-colors/main";
 @import "o-loading/main";
 @import "o-typography/main";
+@import "o-normalise/main";
```

Comment:
Introducing a new dependancy on `o-normalise` is a breaking change as it could cause conflicts for build service users. If you wouldn't mind duplicating the content of `oNormaliseVisuallyHidden` instead, we can create an issue to update this with `o-normalise` in the next major release. :)